funnel/templates/notifications/comment_report_received_email.html.jinja2

```diff
@@ -1,14 +1,18 @@
-{%- extends "notifications/layout_email.html.jinja2" %}
+{%- extends "notifications/layout_email.html.jinja2" -%}
+{%- from "notifications/macros_email.html.jinja2" import cta_button -%}
 {%- block content -%}
 
-  <p>
-    {%- trans %}A comment has been reported as spam{% endtrans -%}
-  </p>
+    <tr align="center">
+        <td class="text-content">
+            <p class="body-content">{%- trans %}A comment has been reported as spam{% endtrans -%}</p>
+        </td>
+    </tr>
+    <br>
 
-  <p class="btn-bar">
-    <a class="btn" href="{{ url_for('siteadmin_review_comment', report=view.report.uuid_b58) }}">
-      {%- trans %}Review comment{% endtrans -%}
-    </a>
-  </p>
+    {# Button : BEGIN #}
+    {{ cta_button(url_for('siteadmin_review_comment', report=view.report.uuid_b58), gettext("Review comment") )}}
+    {# Button : END #}
+
+    {# Email body : END #}
```

Comment:
{# Email body : BEGIN #} is missing. In few other templates too

Reply:
Removed this from the templates since its already part of  `layout`